backend/api/tests/schema/conference/test_ranking.py

```diff
@@ -1,23 +1,30 @@
-from pytest import mark
+import pytest
 
-pytestmark = mark.django_db
+pytestmark = [pytest.mark.django_db]
 
 
-def test_conference_ranking_does_not_exists(conference_factory, graphql_client):
+@pytest.fixture
+def mock_users(mocker):
```

Comment:
uhm ideally we should mock with this `respx` 🤔

Reply:
I switched to `respx`